charts/clearml/values.yaml

```diff
@@ -281,6 +281,8 @@ agentGroups:
       repository: "ubuntu"
       pullPolicy: IfNotPresent
       tag: "18.04"
+    
```

Comment:
Lint is not passing due trailing spaces, can you fix it pls?

Reply:
Fixed it. Sorry I run `helm lint chars/clearml` but it didn't report any error.